packages/utils/src/actions/index.js

```diff
@@ -12,6 +13,7 @@ module.exports = {
   listForApp,
   listForOrg,
   matchForApp,
+  matchManyForApp,
```

Comment:
What's the distinction here? `matchForApp` and `matchManyForApp` both return multiple results. These should be consolidated.